CHANGELOG.md

```diff
@@ -1,5 +1,47 @@
 # Changelog
 
+### 1.0.11 - Wednesday 8 April 2020
+
+- Users can now change password in /setup
+
+![password change](https://posthog.com/wp-content/uploads/2020/04/Posthog-3.png)
+
+- Users can also reset password at login screen
+- Added a logout button
+
+![logout button](https://posthog.com/wp-content/uploads/2020/04/logoutbuton.gif)
+
+- Added GitHub / GitLab Social Authorization
+
+![social auth](https://posthog.com/wp-content/uploads/2020/04/Posthog-1.png)
+
+- Added Stickiness explanation in /trends > Shown As > Stickiness
+
+![Stickiness explanation](https://posthog.com/wp-content/uploads/2020/04/Posthog-4.png)
+
+- Added Celery background workers
```

Comment:
could we move this to top, saying:

`- Important! We've added Celery workers. We'll move a lot of workload off to them to speed up a lot of actions in PostHog. [See update instructions](https://docs.posthog.com/#/upgrading-posthog?id=upgrading-from-before-1011) on how to enable workers.